### PR TITLE
Specify broadcast behavior for Point #239

### PR DESCRIPTION
**Pull request by kylebeggs**
Previously, you could not subtract a single Point from an array of Points using broadcasting. This PR fixes that (Issue #239).

### CODE REVIEW

File: src/points.jl [hunk 1]
```diff
@@ -60,6 +60,9 @@ const Point1f = Point{1,Float32}
 const Point2f = Point{2,Float32}
 const Point3f = Point{3,Float32}
 
+# Enable broadcasting behavior for Point
+Base.Broadcast.broadcastable(p::Point) = Ref(p)
```
Comment (juliohm) [inline src/points.jl]:
```suggestion
# enable broadcasting behavior for Point
Broadcast.broadcastable(p::Point) = Ref(p)
```

**Comment by juliohm** [comment]
Thank you @kylebeggs , can you please add some tests?

**Comment by kylebeggs** [comment]
Ok, should be good now. Still trying to get a full grip on PRs and merging, etc. with git..

File: src/points.jl [hunk 2]
```diff
@@ -60,6 +60,9 @@ const Point1f = Point{1,Float32}
 const Point2f = Point{2,Float32}
 const Point3f = Point{3,Float32}
 
+# enable broadcasting behavior for Point
```
Comment (juliohm) [inline src/points.jl]:
```suggestion
# broadcast behavior
```

**Pull request merged**